sdk/communication/azure-communication-chat/azure/communication/chat/_chat_thread_client.py

```diff
@@ -117,6 +119,32 @@ def thread_id(self):
         """
         return self._thread_id
 
+    @distributed_trace
+    def get_properties(
+        self,
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> ChatThreadProperties
+        """Gets a chat thread properties.
+
+        :return: ChatThreadProperties
+        :rtype: ~azure.communication.chat.ChatThreadProperties
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
```

Comment:
Under what circumstances is `ValueError` raised? Unless it's something specific that users need to be aware of and code for, we can just remove this.

Reply:
It was a miss. It does not throw ValueError in any scenario. This sneaked in when transferring the method from chat_client :)